lib/internal/structured_clone.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const {
+  codes: {
+    ERR_INVALID_ARG_TYPE,
+    ERR_MISSING_ARGS,
+  },
+} = require('internal/errors');
+const webidl = require('internal/webidl');
+
+const {
+  structuredClone: nativeStructuredClone,
+} = internalBinding('messaging');
+
+webidl.converters['sequence<object>'] = webidl.createSequenceConverter(webidl.converters.any);
+
+function structuredClone(value, options) {
```

Comment:
It doesn't look like this is enough to be in its own file which could incur additional compilation cost.  `lib/internal/worker/js_transferable.js` might be a better place for this.